### PR TITLE
Add Enumerables section

### PR DESCRIPTION
**Pull request by joshforisha**

### CODE REVIEW

**Comment by notmessenger** [comment]
There are code examples of enums in both of the files in this repo.  Please update them to reflect the desired format.

**Comment by notmessenger** [comment]
Would this explanation be more appropriate in https://github.com/softlayer/ember-style-guide/blob/master/javascript.md rather than the Ember.js specific guide?

**Comment by joshforisha** [comment]
Updated PR branch to resolve both questions from above.

File: javascript.md [hunk 1]
```diff
@@ -50,6 +50,14 @@ interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt)
 * [Function Arguments](#function-arguments)
 
 
+## Enumerables
+
+* [Enumerables](#enumerables)
+
+
+---
```
Comment (notmessenger) [inline javascript.md]: ?

Comment (joshforisha) [inline javascript.md]: Separator between the Table of Contents and the first section of content.

**Comment by joshforisha** [comment]
Resolves #13

File: javascript.md (hunk 1 above)
Comment (notmessenger) [inline javascript.md]: Oh, I'm seeing why you thought we might need this.  The "Enumerables" heading should have `###` rather than `##`

Comment (joshforisha) [inline javascript.md]: Yes, it should, but that's unrelated to my adding the separator.

**Pull request merged**